docker/web/Dockerfile

```diff
@@ -29,4 +30,4 @@ COPY --from=build /cbioportal/portal/target/dependency/webapp-runner.jar /webapp
 RUN mkdir -p $PORTAL_WEB_HOME
 RUN cd $PORTAL_WEB_HOME && jar -xvf /app.war
 
-CMD ["/bin/sh", "-c", "/usr/bin/java ${JAVA_OPTS} -jar /webapp-runner.jar ${WEBAPP_OPTS} ${PORTAL_WEB_HOME}"]
```

Comment:
`/usr/bin/java` is no longer the default java location. We should let the system decides rather.